app/src/main/java/org/mozilla/focus/telemetry/GleanMetricsService.kt

```diff
@@ -0,0 +1,103 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.telemetry
+
+import android.content.Context
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers.IO
+import kotlinx.coroutines.MainScope
+import kotlinx.coroutines.async
+import kotlinx.coroutines.launch
+import mozilla.components.browser.state.search.SearchEngine
+import mozilla.components.browser.state.state.selectedOrDefaultSearchEngine
+import mozilla.components.feature.search.ext.waitForSelectedOrDefaultSearchEngine
+import mozilla.components.service.glean.net.ConceptFetchHttpUploader
+import mozilla.telemetry.glean.Glean
+import mozilla.telemetry.glean.config.Configuration
+import org.mozilla.focus.BuildConfig
+import org.mozilla.focus.Components
+import org.mozilla.focus.GleanMetrics.GleanBuildInfo
+import org.mozilla.focus.GleanMetrics.Browser
+import org.mozilla.focus.GleanMetrics.LegacyIds
+import org.mozilla.focus.GleanMetrics.Pings
+import org.mozilla.focus.ext.components
+import org.mozilla.focus.telemetry.TelemetryWrapper.isTelemetryEnabled
+import org.mozilla.focus.utils.Settings
+import java.util.UUID
+
+class GleanMetricsService(private val context: Context) : MetricsService {
```

Comment:
I would really recommend not going through this and not following the Fenix example on this implementation, if possible. This proved to be hard to debug, a performance bottleneck and in general problematic. Instead of wrapping Glean, would you consider calling the Glean APIs directly?

Reply:
Using Glean directly makes it difficult to unit test components that don't need to know about it.

I don't think we need to introduce the [`MetricsController`](https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/java/org/mozilla/fenix/components/metrics/MetricController.kt) abstraction that Fenix uses which is definitely harder to use, debug and increases perf complications.

Reply:
> 
> 
> Using Glean directly makes it difficult to unit test components that don't need to know about it.

You got my interest :-) Do you have an example about this? Could you file a bug? We'd love to hear about this case and fix it.

Glean should be usable without causing any problem anywhere in the codebase.

Reply:
I tried to reproduce some old behaviour and we no longer crash in Fenix without needing to mock it out! 🙌 

We try to reduce the need for instrumented tests (AndroidJUnit/Robolectric) tests because of how slow they are, but we can write separate unit tests that where needed.

I've filed issues [for outstanding effects](https://bugzilla.mozilla.org/show_bug.cgi?id=1706682); will file more if they come up. 🙂

Reply:
Heh, maybe I spoke too soon. Focus tests fail when `registerPing` is called during Application#onCreate. Adding the GleanTestRule doesn't help since it doesn't mock out Glean code. I'll file a bug for this too:

```
Native library (com/sun/jna/darwin/libjnidispatch.jnilib) not found in resource path (/Users/jalmeida/.gradle/caches/6.5.1/workerMain/gradle-worker.jar:/Users/jalmeida/src/focus-android/app/build/intermediates/unit_test_config_directory/focusDebugUnitTest/out:/Users/jalmeida/src/focus-android/app/build/intermediates/javac/focusDebugUnitTest/classes:/Users/jalmeida/src/focus-android/app/build/tmp/kotlin-classes/focusDebugUnitTest:/Users/jalmeida/src/focus-android/app/build/intermediates/java_res/focusDebugUnitTest/out:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.adjust.sdk/adjust-android/4.11.4/5841f36e345fe24705ee80da9ff1764769fac7a6/adjust-android-4.11.4.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/f1001693b60d1e89c103b6c6164b9700/installreferrer-1.1-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/46563c81935228461a949eedee3f5e91/feature-contextmenu-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/4554c193b345431e425798fe3ba3bdcd/feature-app-links-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/86be3756298713bc71caaf2d49c574b4/feature-customtabs-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/2fd64f9c7bc04190863e237726685489/feature-intent-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/68999b73f911b67980586c4084bf9a8a/feature-search-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/643125d7fe81d53f3350186a01b181ad/feature-tabs-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/26e9abe46ff6da494eb16a810145260e/feature-session-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/b0e6c81bdb2d9f297f640b4acda5789e/browser-session-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/c917a6d7bd835227d0b5fb4afec27bde/feature-downloads-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/217e156c067d87a17d2ce4b9beb10976/feature-findinpage-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/6716f7b3b93c993b166fe4503baf4c77/feature-prompts-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/d30618e133a0a1db9bc488b6c6e82c4c/browser-toolbar-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/d7b1ffc51efe235da94caf6d5b372350/browser-menu-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/a240b510fe48928c5084ad17d59c2872/browser-session-storage-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/c3f533d02d32497aacc261748aef380a/browser-thumbnails-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/25ef1e3468c2483319ed6b7df698f031/browser-state-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/666c54ed8e8f407ab8aa8e776a863051/browser-engine-gecko-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/649de2fcda1128b4078891e8d2cb4761/concept-tabstray-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/03b8dabb2b89ce38e8ac14cfe6939c13/concept-engine-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/b02d85e1b296d91e477f6f43ffa41b8d/lib-crash-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/821b6a7e95e9a30c27371085531c22f6/lib-state-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/121a7b72e878302d5f69270ad3b03466/service-glean-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/d8d25acfe7ec536cfde02a11c7d7d35f/service-location-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/b6fac10a649c0a5b012ba88daab285a4/browser-errorpages-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/0131a545ead222a44b3bce62510cce9a/ui-tabcounter-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/be3ef7d5c20b49f9c64997b3f0f024b5/browser-menu2-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/43fb986070ee3d8dcf8d5828945d23d1/concept-menu-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/1056741b374881e23e64502ff96fa467/service-digitalassetlinks-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/7ecabce39d60fe580bc848d04624b82d/concept-toolbar-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/45d4f6ea9a6553d18fd011aa4933ffe9/support-ktx-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/5502620d81a61cb587b93043ef78858f/ui-autocomplete-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/72e74a4bdeb94dbbace01db52b82578d/support-utils-90.0.20210525143110-runtime.jar:/Users/jalmeida/src/focus-android/service-telemetry/build/intermediates/runtime_library_classes_jar/debug/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/9d6fd05738ad3e56617db5d378aebb9a/support-images-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/60492f8421a8ee43095ad7ecf9f2517a/support-base-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/828a5d56b1c9cdacf292a95fb159b1a0/concept-storage-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/83179e064ef979d6400ddbd177fcd80a/glean-38.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/2b4932b83d83f397c3833e840f4be942/concept-base-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-android-extensions-runtime/1.4.30/63eabf4e818102cd4e435c6b464762d06abbf754/kotlin-android-extensions-runtime-1.4.30.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/2c502aa20f3468a1572786dad131c01c/preference-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/94bc1b202d6ab2ba209aa2e7713dc1b2/material-1.2.1-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/7b645f14ee40d06d70551c7b9c86691e/constraintlayout-2.0.4-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/137efbb4b587f425823ed433773911f4/appcompat-1.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/756fa6d8b3181b086df278024fd17c3e/browser-1.3.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/1fe624db448c514dc59817bf1a6ab3fe/core-ktx-1.3.2-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/fdc267cffa3c23458aa23cf2c166abaa/cardview-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/99ca8087bdc1e8eaca2c46039cdccc66/paging-runtime-2.1.2-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/ac2d0f5136bb0b15547fe9d9939378b6/viewpager2-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/a97e70b68b09a51008835368d6538486/recyclerview-1.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/ce0530118f0f1267240a10c376cb5a45/geckoview-nightly-90.0.20210525093431-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/344c43aae75a4fda8bca5b161193bf94/legacy-support-v4-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/io.sentry/sentry-android/1.7.21/9d40b2feefd256bd5dad98e78d808c28f0913ef0/sentry-android-1.7.21.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/7de7fd6a58bc33b6b21466f5f8869147/lifecycle-extensions-2.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.robolectric/robolectric/4.4/418c5bfae392fdbf71cd463a42a3e8c3b839a924/robolectric-4.4.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.robolectric/shadows-framework/4.4/90028766e71353ad6f57d7bcb56ac0d861da18c3/shadows-framework-4.4.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.robolectric/resources/4.4/a2ee1324bcb62724e6cbfa655bdb5683948a554c/resources-4.4.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.robolectric/junit/4.4/fbcda51d8e6f3a3897ae5cedc7aa481815745290/junit-4.4.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.robolectric/sandbox/4.4/3cedd73c5aedaf79fb9a593552816c9fb3282f2/sandbox-4.4.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.robolectric/utils-reflector/4.4/44c40ac0d2ef1e7c8b0f6c4e224ef26d356170f1/utils-reflector-4.4.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.robolectric/plugins-maven-dependency-resolver/4.4/9241a3c4bd01627447c76d9b67614808c78ffdd9/plugins-maven-dependency-resolver-4.4.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.robolectric/utils/4.4/c54b2638d64e7bd4e1e45c4fe8038305402bd711/utils-4.4.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.google.guava/guava/27.0.1-jre/bd41a290787b5301e63929676d792c507bbc00ae/guava-27.0.1-jre.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.google.code.findbugs/jsr305/3.0.2/25ea2e8b0c338a877313bd4672d3fe056ea78f0d/jsr305-3.0.2.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/04ffa5c7c129df6e4ebe54de89cc2ae7/browser-domains-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/caab06679735e18d5f38ee7cbe7bb1b9/lib-fetch-okhttp-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/fa193f4f9e03f5c1198c2c31c46bb380/concept-fetch-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/502c2bfcf2e0b3705c6f3a059147d1b2/ui-colors-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.squareup.okhttp3/mockwebserver/3.11.0/df5ba87db2751d4076358ba6ad63045b031c8fd3/mockwebserver-3.11.0.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.squareup.okhttp3/okhttp-urlconnection/3.13.1/1bc9e51426e088b449f40a767e0c45c3baa910f3/okhttp-urlconnection-3.13.1.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.squareup.okhttp3/okhttp/3.14.1/67612a22d4b8f33c55263b188bf5a72774d06d18/okhttp-3.14.1.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jdk8/1.4.30/aeecfab2e6acbcb80fc6628e8882c98cbbd819cf/kotlin-stdlib-jdk8-1.4.30.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jdk7/1.4.30/cd81ed90fd79e43f587bd384c2bc6d1aed0bfe31/kotlin-stdlib-jdk7-1.4.30.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/e0b6eaa95ba01480767b49d629cd39fc/lifecycle-runtime-ktx-2.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/9b36eca580ed3ac5e40137955455f78c/room-ktx-2.2.5-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/05c9754dd1ccc8874aed33f8e40dfd9a/work-runtime-ktx-2.4.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-android/1.4.2/dbf36cc9f9cc03a72e20bf440cfeb89d74ca775f/kotlinx-coroutines-android-1.4.2.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/32caf75a1689328f1a738dd31a66e89c/palette-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter-params/5.3.1/9dfac3fbd6768974fc0c142304a3e90ba713b2a8/junit-jupiter-params-5.3.1.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter-engine/5.3.1/e2676b1786c57a80eb98f5bebd51a3d05e228c40/junit-jupiter-engine-5.3.1.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter-api/5.3.1/a7e97eac2784395cb991403f9641b042ad972941/junit-jupiter-api-5.3.1.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/2b1aed496448f118443ed5cc49783e33/core-testing-2.1.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/d788ba06e90cbbafee6dc6a38d4cb42d/rules-1.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/dc15269e58155ba29335f1db768dc4f2/runner-1.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.junit.vintage/junit-vintage-engine/5.3.1/48e21c1c1a7c1dde68c2f71181b3e86927c7923e/junit-vintage-engine-5.3.1.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/junit/junit/4.12/2973d150c0dc1fefe998f834810d68f278ea58ec/junit-4.12.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.mockito/mockito-core/2.24.5/599509fe319bd9e39559b8f987bee5d4b77167e4/mockito-core-2.24.5.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/cc4fbfba7f5a5eff2c7e366753d30e83/core-1.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/1563431e3c2c499c05587eb28e4c59e3/lifecycle-livedata-ktx-2.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/43a3927e1f6bff9b89834f9bba3715cb/lifecycle-livedata-core-ktx-2.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-core-jvm/1.4.2/4b9c6b2de7cabfb2c9ad7a5c709b1ddb7bbfd2ad/kotlinx-coroutines-core-jvm-1.4.2.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.4.30/d10d1e10f47006ee08162dde039e38ac487de4ac/kotlin-stdlib-1.4.30.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/93d6b82da9a85f9b3055d66a54e24635/legacy-support-core-ui-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/d93f5ca9ebd580feb9b2009b3f9c263f/legacy-support-core-utils-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/18711bc3a78085a287a5e5cf4c7b4214/media-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/b3481fbc7a4fc22b4962a0747292c26b/work-runtime-2.4.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/cb660cc123eb7fe2c79cd64b552d16f9/room-runtime-2.2.5-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/984312a471fc1eedc496337581e434d9/play-services-fido-18.1.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/512796b7bfb9664878f76e92b599f8ee/play-services-base-17.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/ebcbf2c20cf1ccaeb40424cfaacd4ab2/play-services-tasks-17.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/5723c50eee43d8b0e3d8a4cb2afa8985/play-services-basement-17.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/a0241c93ff4cb9f4aa71463066ec7bec/fragment-1.2.5-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/6d22062723de33e553ffb9745dd457e6/loader-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/c4aee73f11866a8ffede6a76c98b96c8/lifecycle-livedata-2.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/1780b302a2be3d59bbf022cce5b4fd15/activity-1.1.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/a9670e9ad558b00259a2ad6425718576/lifecycle-viewmodel-savedstate-2.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/e413433c3088bcf3ef885ba22433045a/lifecycle-livedata-core-2.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/467b187f0dc0dd449cbb2bbde31898ec/core-runtime-2.1.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/01260b6e2c7e8ab7c643392e066ea1fe/lifecycle-process-2.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/e2b7f7db0e689b90712b6b56029b35c1/swiperefreshlayout-1.1.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/ecde404a1099c2eff1fdd3fdd47521f1/viewpager-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/3e205ea674ce67741c7de0708907af1c/drawerlayout-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/9b256e1ba1f6cf0072277c8164ae67fd/slidingpanelayout-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/77eb5a9900498df4081a158e5863d70f/coordinatorlayout-1.1.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/081e591ffe533d9182f4fb7ebf3fa370/customview-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/6a77f855790766299f2e7b6e26149351/asynclayoutinflater-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/b872837d60b5033d436eeadf4f3a5d78/transition-1.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/c79cf521201e5d06fb87ff41151f600d/appcompat-resources-1.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/098b3f4f61704960aebd5ee2c6cd9cf6/vectordrawable-animated-1.1.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/6ee184a70ecec6b43045d1c16642fbb4/vectordrawable-1.1.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/d036d6a7fcde929bc5c0ab7bf1d8dd8c/core-1.3.2-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/0c2de9fedbaa38015faf7d245c92b029/lifecycle-service-2.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/1aaa4fae5a5cd85f0995f56d3f4fa70b/lifecycle-runtime-2.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/androidx.paging/paging-common/2.1.2/e8cdd44767173c4af5164732c32741c89afa684d/paging-common-2.1.2.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/87313bba480ea5b3ca5257ac3fe98887/savedstate-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/androidx.arch.core/core-common/2.1.0/b3152fc64428c9354344bd89848ecddc09b6f07e/core-common-2.1.0.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/5a3bad67327d80bbdaa8d2aaf3584dba/localbroadcastmanager-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/09b29464744bb3e631d53e87320932bb/monitor-1.3.0-rc03-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/a9b02cc9059a03b4fd1660af347737ec/documentfile-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/4b0effffe869956fda9fb2e1518b237c/print-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/fef930b32f9564dccacce8fc01a838b3/interpolator-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/9d7c7fa1ca625458caa5ca0778c3e0bc/cursoradapter-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/androidx.lifecycle/lifecycle-common-java8/2.0.0/52cb9efc663d8bebc82f8979a7da30938bd05611/lifecycle-common-java8-2.0.0.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/491766e7c251e8aedc260cf5b219001c/versionedparcelable-1.1.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/androidx.collection/collection/1.1.0/1f27220b47669781457de0d600849a5de0e89909/collection-1.1.0.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/androidx.concurrent/concurrent-futures/1.0.0/c1e77e3ee6f4643b77496a1ddf7a2eef1aefdaa1/concurrent-futures-1.0.0.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/androidx.room/room-common/2.2.5/f5e3b73a0c2ab5e276e26868e4ce3542baede207/room-common-2.2.5.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/3a815b8b196052753ff4a272261ffb14/lifecycle-viewmodel-2.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/androidx.lifecycle/lifecycle-common/2.2.0/4ef09a745007778eef83b92f8f23987a8ea59496/lifecycle-common-2.2.0.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/30a69b4e07c17315b815ba9bee2115e9/sqlite-framework-2.1.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/c347637f4c66eec533f4ef33946c2bfb/sqlite-2.1.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/androidx.annotation/annotation/1.1.0/e3a6fb2f40e3a3842e6b7472628ba4ce416ea4c8/annotation-1.1.0.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/io.sentry/sentry/1.7.21/830b5214d178189de482443aa20919697079edfd/sentry-1.7.21.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/778523868023ece6cb8b09593bf392ee/ui-icons-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.squareup.okio/okio/1.17.2/78c7820b205002da4d2d137f6f312bd64b3d6049/okio-1.17.2.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-engine/1.3.1/3ee68a06bbdab157dd260e2095c356481d6cd172/junit-platform-engine-1.3.1.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-commons/1.3.1/67b7edddfac1935e6e6d9b58d7c7df6db59b1d39/junit-platform-commons-1.3.1.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.apiguardian/apiguardian-api/1.0.0/3ef5276905e36f4d8055fe3cb0bdcc7503ffc85d/apiguardian-api-1.0.0.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.opentest4j/opentest4j/1.1.1/efd9f971e91074491ea55b19f67b13470cf4fcdd/opentest4j-1.1.1.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.hamcrest/hamcrest-core/1.3/42a25dc3219429f0e5d060061f71acb49bf010a0/hamcrest-core-1.3.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.robolectric/pluginapi/4.4/1ee94260f8c51620a35eac33fc1efc01350c751f/pluginapi-4.4.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.robolectric/shadowapi/4.4/529649474b53cf8f6f4a483044ade43aebed8a4c/shadowapi-4.4.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.robolectric/annotations/4.4/70fc5b1699467dfd7de606fc6c02ff9fc1816d9f/annotations-4.4.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/javax.inject/javax.inject/1/6975da39a7040257bd51d21a231b76c915872d38/javax.inject-1.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/javax.annotation/javax.annotation-api/1.3.2/934c04d3cfef185a8008e7bf34331b79730a9d43/javax.annotation-api-1.3.2.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.bouncycastle/bcprov-jdk15on/1.65/320b989112f00a63a3bcfa5a98f31a4f865a20fa/bcprov-jdk15on-1.65.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/net.bytebuddy/byte-buddy/1.9.7/8fea78fea6449e1738b675cb155ce8422661e237/byte-buddy-1.9.7.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/net.bytebuddy/byte-buddy-agent/1.9.7/8e7d1b599f4943851ffea125fd9780e572727fc0/byte-buddy-agent-1.9.7.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.objenesis/objenesis/2.6/639033469776fd37c08358c6b92a4761feb2af4b/objenesis-2.6.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/net.sf.kxml/kxml2/2.3.0/ccbc77a5fd907ef863c29f3596c6f54ffa4e9442/kxml2-2.3.0.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-common/1.4.30/bb9a3173350f55732416ee27956ea8f9b81f4dbb/kotlin-stdlib-common-1.4.30.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.jetbrains/annotations/13.0/919f0dfe192fb4e063e7dacadee7f8bb9a2672a9/annotations-13.0.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.slf4j/slf4j-api/1.7.24/3f6b4bd4f8dbe8d4bea06d107a3826469b85c3e9/slf4j-api-1.7.24.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-core/2.8.7/8b46f39c78476fb848c81a49fa807a9e9506dddd/jackson-core-2.8.7.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.yaml/snakeyaml/1.24/13a9c0d6776483c3876e3ff9384f9bb55b17001b/snakeyaml-1.24.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.google.guava/listenablefuture/9999.0-empty-to-avoid-conflict-with-guava/b421526c5f297295adef1c886e5246c39d4ac629/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/80e35b9b2b9bcec7600a88317a74bb79/annotation-experimental-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/androidx.constraintlayout/constraintlayout-solver/2.0.4/1f001d7db280a89a6c26b26a66eb064bb6d5efeb/constraintlayout-solver-2.0.4.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.jakewharton/disklrucache/2.0.2/93b399470e72c0bc8cb52379943b63c9aa586cd8/disklrucache-2.0.2.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/1bef3144ed1b8e5e8ac447bf5cb5f151/jna-5.6.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-commons/7.2/ca2954e8d92a05bacc28ff465b25c70e0f512497/asm-commons-7.2.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-util/7.2/a3ae34e57fa8a4040e28247291d0cc3d6b8c7bcf/asm-util-7.2.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-analysis/7.2/b6e6abe057f23630113f4167c34bda7086691258/asm-analysis-7.2.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-tree/7.2/3a23cc36edaf8fc5a89cb100182758ccb5991487/asm-tree-7.2.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm/7.2/fa637eb67eb7628c915d73762b681ae7ff0b9731/asm-7.2.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.almworks.sqlite4java/sqlite4java/0.282/745a7e2f35fdbe6336922e0d492c979dbbfa74fb/sqlite4java-0.282.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.ibm.icu/icu4j/53.1/786d9055d4ca8c1aab4a7d4ac8283f973fd7e41f/icu4j-53.1.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.google.auto.value/auto-value-annotations/1.6.2/ed193d86e0af90cc2342aedbe73c5d86b03fa09b/auto-value-annotations-1.6.2.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.google.guava/failureaccess/1.0.1/1dcf1de382a0bf95a3d8b0849546c88bac1292c9/failureaccess-1.0.1.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.checkerframework/checker-qual/2.5.2/cea74543d5904a30861a61b4643a5f2bb372efc4/checker-qual-2.5.2.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.google.errorprone/error_prone_annotations/2.2.0/88e3c593e9b3586e1c6177f89267da6fc6986f0c/error_prone_annotations-2.2.0.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.google.j2objc/j2objc-annotations/1.1/ed28ded51a8b1c6b112568def5f4b455e6809019/j2objc-annotations-1.1.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.codehaus.mojo/animal-sniffer-annotations/1.17/f97ce6decaea32b36101e37979f8b647f00681fb/animal-sniffer-annotations-1.17.jar:/Users/jalmeida/src/focus-android/app/build/intermediates/compile_and_runtime_not_namespaced_r_class_jar/focusDebug/R.jar:/Users/jalmeida/src/focus-android/app/build/intermediates/javac/focusDebug/classes:/Users/jalmeida/src/focus-android/app/build/tmp/kotlin-classes/focusDebug:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/31e56a097e4fc62a0ffa1fc11d089069/installreferrer-1.1/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/82d4b96b33befad79512c4299964c971/feature-contextmenu-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/9707f492c3537e0afb0e256fc76e30db/feature-app-links-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/0adb53459cd1450dff6140454386d1e6/feature-customtabs-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/cc3a4ee876613805cfbcc4e95e5f1a65/feature-intent-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/1c3e82fb01f6736dc02bdae6614048a7/feature-search-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/9563790f7ba2c314616c89f75f36aeac/feature-tabs-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/55a45db949e21619bdc899c1d8f23917/feature-session-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/e3c6f678b06c7dd9b497f487967fd568/browser-session-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/1e8a5c7b290e38815a090b044438bfc5/feature-downloads-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/41862494f2a5f79e617f8241ab7dfd24/feature-findinpage-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/8e455adb9c6f72f1448b76ad710bd8b5/feature-prompts-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/511d5c73ac2e91208a56bb91f14c4400/browser-toolbar-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/43abbfde3490f0d7fc8dcb24044c0957/browser-menu-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/d486296659bbaccc0770cef48bcbb706/browser-session-storage-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/4003059466598910d1aa18f44dd5bfcd/browser-thumbnails-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/a343424ebdbcfbb300d4109d63a09f34/browser-state-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/0b8844e28df41adbccf5fe2af1ea3421/browser-engine-gecko-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/6738156cc38ed9619bf152f5c68c2e76/concept-tabstray-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/252e14b4f1a0c5f5ad56c1cc9df0104d/concept-engine-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/e9c654544070f9f37237a814c9a1986d/lib-crash-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/9ea84685adfaedb2717d1cfea48a0900/lib-state-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/cce0af0cc74747849858e02af0fc9f6b/service-glean-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/863d95537d20bc9cd7d096c0ddab4fa0/service-location-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/609540216dffda8a953e2792a7b7151c/browser-errorpages-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/fc8afe27905acfc133d1521a8536e4ee/ui-tabcounter-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/881b48c91ee320ba3671cc489255c412/browser-menu2-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/05a2432c97263acd52860bb693638562/concept-menu-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/4580f7385ef8073c7093b50b636614e5/service-digitalassetlinks-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/ad216dbeedf93cd46fd08b9b36b36160/concept-toolbar-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/daf4bf7588ca4733352b1e0f16901b21/support-ktx-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/17bb3b66f77cec3d57693dd68848b3d1/ui-autocomplete-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/35f11394e4c9361b41471b61278f4ff1/support-utils-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/src/focus-android/service-telemetry/build/intermediates/library_java_res/debug/res.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/0279f60c2a780813133b7330a4f6f661/support-images-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/ec4d8873bb8d8fae34fedcf23bc35c05/support-base-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/a6884f259c1a6e98784c6137f1aa8b14/concept-storage-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/bf50c7a7796b144c85ab243f2584cad6/glean-38.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/cc0f2c8def8e7b3d02a02d96afcb6cfc/concept-base-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/e0b89432673094fe60dd8eca410725ba/preference-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/a77452e49f1b1a02c3fb9f968ee794f8/material-1.2.1/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/5afb07e94434c35f38a4072e015540ad/constraintlayout-2.0.4/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/1cebdd0cb62ee96ba6d82d4eb3252724/appcompat-1.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/ab1a9225d7ac271b545ceae199106624/browser-1.3.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/bd252a46c023f1b6a02aa1bf5ba425a5/core-ktx-1.3.2/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/7602d1e760abf229d4f523249ca44e44/cardview-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/d8566c2c2c13ea3fbd4ef660e85f5076/paging-runtime-2.1.2/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/9707758b67eb0793926d959df28e44bc/viewpager2-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/9a62a70274a70a80baadf3dfb668583a/recyclerview-1.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/22786bb75c49819f7d927d0cd4139184/geckoview-nightly-90.0.20210525093431/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/94a411c830ccfe277dd49a59e234f3f3/legacy-support-v4-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/b7cb06fdfe6e1124f5ab63806fc2d08e/lifecycle-extensions-2.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/9e8a892eef9149dd5a398891e15869a0/browser-domains-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/5510a99df70ab24e45f3dbff572a76f9/lib-fetch-okhttp-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/6649495d6f9e484ee388f3bff2b1dc14/concept-fetch-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/ceab4852ca75320d223c1fd444630644/ui-colors-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/6db00be38e8f5bcbb91e75774bdaea3d/lifecycle-runtime-ktx-2.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/e752397819bc17ef70419c54faea4cbc/room-ktx-2.2.5/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/f57e5567ba8179e752178d774f3f0e69/work-runtime-ktx-2.4.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/db39f25969e1df57f8a7c83999856e92/palette-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/cb75a1d2bfa101fbdcea7ecc7f454830/core-testing-2.1.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/55e92f6e25a6bb6f780651a499ec38aa/rules-1.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/1c7e44a41cb9952becc507daa816dcca/runner-1.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/86bc51f9344de2c733faad65ea4d12e3/core-1.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/8b91830be8a7fd3e5593f93a5884eeab/lifecycle-livedata-ktx-2.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/06a311992c275806754eafcbcf418722/lifecycle-livedata-core-ktx-2.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/bff5366f96ffdc3661c78657291e0ce7/legacy-support-core-ui-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/b24610df9369ff0969b2658d8450ce4d/legacy-support-core-utils-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/34ac03f8e1c63dc0859f87211b34c5d5/media-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/ac43a4b99ed8a4f71c3019bfdcd79c4c/work-runtime-2.4.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/8e03a9e366a2934f9befefd10c602917/room-runtime-2.2.5/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/c4d75a66d7693ec93a662a871c9a6845/play-services-fido-18.1.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/21e5aecd7492c0b8ef2fbc14b40b0eb0/play-services-base-17.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/12c81bcc65cb60319de0670fe274c572/play-services-tasks-17.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/93b6a12158fbe121d0710c9cd4160f42/play-services-basement-17.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/147175a6c56bc4e6e494b5265e007a4f/fragment-1.2.5/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/20188848923738ceb5ad11de5c5200b1/loader-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/ed2461abcd77b2425b399947ab2b1042/lifecycle-livedata-2.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/5540ac8b4098750209eba4fb31d834db/activity-1.1.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/0e20ad0108f5fb86d7cbe0c06b2d9ace/lifecycle-viewmodel-savedstate-2.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/b8d6d22b7d7ddf2668e4a9dd1acd1467/lifecycle-livedata-core-2.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/08362102e5bf5d5150c1ecc8db5f9e8c/core-runtime-2.1.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/6169c8b80960f217ff7b9626683531bd/lifecycle-process-2.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/d252d13fd7b33bb81ee566ce91b028c6/swiperefreshlayout-1.1.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/36de97678321d5eadd77afef7f67810c/viewpager-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/7640c04ce6d80049852812ee3aa6851d/drawerlayout-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/b76a316a99f7475c690654fe178a0fdb/slidingpanelayout-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/9dcf5f5c72568de51fdf8bd9a358e472/coordinatorlayout-1.1.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/78469aee7b254965b30e1956bfcbb25e/customview-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/5ac3ef0ca50830e4fa0e397329b286de/asynclayoutinflater-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/be0b8eb2d44cc1ccfbeae4480320e828/transition-1.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/79376807bce34fbd136eb820f35b8cd6/appcompat-resources-1.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/65e0da78bb9613fd85c1ee7149834ddf/vectordrawable-animated-1.1.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/816905beec122a9212c673214254a9c9/vectordrawable-1.1.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/7325fbff668234181bfb596cf97854d8/core-1.3.2/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/04ef2866830ad118bf92f4cf3e9d91a5/lifecycle-service-2.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/8894fd351bfc682f394ffc6fdf7c016c/lifecycle-runtime-2.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/ea2a9331c37d9f6d11758b5e50bbfa27/savedstate-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/30f2ce583cdb1e061e2d70c70f55c4c5/localbroadcastmanager-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/d2e93e7a0c6e8fda2a5d657b6a963ee5/monitor-1.3.0-rc03/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/839c7bb928a6a087500b514c0161ff97/documentfile-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/b62b5a7b68556024837bd4c5bac5078c/print-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/0cbc3bbf668d2a804bca0284e59ed477/interpolator-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/c2fd7ca49c0f447223c66b875c2b40fd/cursoradapter-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/adf81b82f94a0cd15e3db709beb6af60/versionedparcelable-1.1.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/3e27cfdac886e7b9d1eeaf437b695a74/lifecycle-viewmodel-2.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/6a40dc0ca777ecbec5ddeb18ecb8b901/sqlite-framework-2.1.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/8c5edc9e1e8e4f58b71019f63901edef/sqlite-2.1.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/5bec7fdc24bc3e01917a19dd0ad49afd/ui-icons-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/dcc6b72b51bd90a58c813a03f8ba9546/annotation-experimental-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/29509c42b190e288c362e63e49a0b4b3/jna-5.6.0/jars/classes.jar:/Users/jalmeida/src/focus-android/app/build/intermediates/java_res/focusDebug/out:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/f5e15ff62d155b5e1b4fe2da328bb96c/android.jar)
java.lang.UnsatisfiedLinkError: Native library (com/sun/jna/darwin/libjnidispatch.jnilib) not found in resource path (/Users/jalmeida/.gradle/caches/6.5.1/workerMain/gradle-worker.jar:/Users/jalmeida/src/focus-android/app/build/intermediates/unit_test_config_directory/focusDebugUnitTest/out:/Users/jalmeida/src/focus-android/app/build/intermediates/javac/focusDebugUnitTest/classes:/Users/jalmeida/src/focus-android/app/build/tmp/kotlin-classes/focusDebugUnitTest:/Users/jalmeida/src/focus-android/app/build/intermediates/java_res/focusDebugUnitTest/out:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.adjust.sdk/adjust-android/4.11.4/5841f36e345fe24705ee80da9ff1764769fac7a6/adjust-android-4.11.4.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/f1001693b60d1e89c103b6c6164b9700/installreferrer-1.1-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/46563c81935228461a949eedee3f5e91/feature-contextmenu-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/4554c193b345431e425798fe3ba3bdcd/feature-app-links-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/86be3756298713bc71caaf2d49c574b4/feature-customtabs-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/2fd64f9c7bc04190863e237726685489/feature-intent-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/68999b73f911b67980586c4084bf9a8a/feature-search-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/643125d7fe81d53f3350186a01b181ad/feature-tabs-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/26e9abe46ff6da494eb16a810145260e/feature-session-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/b0e6c81bdb2d9f297f640b4acda5789e/browser-session-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/c917a6d7bd835227d0b5fb4afec27bde/feature-downloads-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/217e156c067d87a17d2ce4b9beb10976/feature-findinpage-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/6716f7b3b93c993b166fe4503baf4c77/feature-prompts-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/d30618e133a0a1db9bc488b6c6e82c4c/browser-toolbar-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/d7b1ffc51efe235da94caf6d5b372350/browser-menu-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/a240b510fe48928c5084ad17d59c2872/browser-session-storage-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/c3f533d02d32497aacc261748aef380a/browser-thumbnails-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/25ef1e3468c2483319ed6b7df698f031/browser-state-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/666c54ed8e8f407ab8aa8e776a863051/browser-engine-gecko-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/649de2fcda1128b4078891e8d2cb4761/concept-tabstray-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/03b8dabb2b89ce38e8ac14cfe6939c13/concept-engine-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/b02d85e1b296d91e477f6f43ffa41b8d/lib-crash-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/821b6a7e95e9a30c27371085531c22f6/lib-state-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/121a7b72e878302d5f69270ad3b03466/service-glean-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/d8d25acfe7ec536cfde02a11c7d7d35f/service-location-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/b6fac10a649c0a5b012ba88daab285a4/browser-errorpages-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/0131a545ead222a44b3bce62510cce9a/ui-tabcounter-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/be3ef7d5c20b49f9c64997b3f0f024b5/browser-menu2-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/43fb986070ee3d8dcf8d5828945d23d1/concept-menu-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/1056741b374881e23e64502ff96fa467/service-digitalassetlinks-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/7ecabce39d60fe580bc848d04624b82d/concept-toolbar-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/45d4f6ea9a6553d18fd011aa4933ffe9/support-ktx-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/5502620d81a61cb587b93043ef78858f/ui-autocomplete-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/72e74a4bdeb94dbbace01db52b82578d/support-utils-90.0.20210525143110-runtime.jar:/Users/jalmeida/src/focus-android/service-telemetry/build/intermediates/runtime_library_classes_jar/debug/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/9d6fd05738ad3e56617db5d378aebb9a/support-images-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/60492f8421a8ee43095ad7ecf9f2517a/support-base-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/828a5d56b1c9cdacf292a95fb159b1a0/concept-storage-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/83179e064ef979d6400ddbd177fcd80a/glean-38.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/2b4932b83d83f397c3833e840f4be942/concept-base-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-android-extensions-runtime/1.4.30/63eabf4e818102cd4e435c6b464762d06abbf754/kotlin-android-extensions-runtime-1.4.30.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/2c502aa20f3468a1572786dad131c01c/preference-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/94bc1b202d6ab2ba209aa2e7713dc1b2/material-1.2.1-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/7b645f14ee40d06d70551c7b9c86691e/constraintlayout-2.0.4-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/137efbb4b587f425823ed433773911f4/appcompat-1.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/756fa6d8b3181b086df278024fd17c3e/browser-1.3.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/1fe624db448c514dc59817bf1a6ab3fe/core-ktx-1.3.2-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/fdc267cffa3c23458aa23cf2c166abaa/cardview-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/99ca8087bdc1e8eaca2c46039cdccc66/paging-runtime-2.1.2-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/ac2d0f5136bb0b15547fe9d9939378b6/viewpager2-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/a97e70b68b09a51008835368d6538486/recyclerview-1.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/ce0530118f0f1267240a10c376cb5a45/geckoview-nightly-90.0.20210525093431-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/344c43aae75a4fda8bca5b161193bf94/legacy-support-v4-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/io.sentry/sentry-android/1.7.21/9d40b2feefd256bd5dad98e78d808c28f0913ef0/sentry-android-1.7.21.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/7de7fd6a58bc33b6b21466f5f8869147/lifecycle-extensions-2.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.robolectric/robolectric/4.4/418c5bfae392fdbf71cd463a42a3e8c3b839a924/robolectric-4.4.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.robolectric/shadows-framework/4.4/90028766e71353ad6f57d7bcb56ac0d861da18c3/shadows-framework-4.4.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.robolectric/resources/4.4/a2ee1324bcb62724e6cbfa655bdb5683948a554c/resources-4.4.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.robolectric/junit/4.4/fbcda51d8e6f3a3897ae5cedc7aa481815745290/junit-4.4.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.robolectric/sandbox/4.4/3cedd73c5aedaf79fb9a593552816c9fb3282f2/sandbox-4.4.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.robolectric/utils-reflector/4.4/44c40ac0d2ef1e7c8b0f6c4e224ef26d356170f1/utils-reflector-4.4.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.robolectric/plugins-maven-dependency-resolver/4.4/9241a3c4bd01627447c76d9b67614808c78ffdd9/plugins-maven-dependency-resolver-4.4.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.robolectric/utils/4.4/c54b2638d64e7bd4e1e45c4fe8038305402bd711/utils-4.4.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.google.guava/guava/27.0.1-jre/bd41a290787b5301e63929676d792c507bbc00ae/guava-27.0.1-jre.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.google.code.findbugs/jsr305/3.0.2/25ea2e8b0c338a877313bd4672d3fe056ea78f0d/jsr305-3.0.2.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/04ffa5c7c129df6e4ebe54de89cc2ae7/browser-domains-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/caab06679735e18d5f38ee7cbe7bb1b9/lib-fetch-okhttp-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/fa193f4f9e03f5c1198c2c31c46bb380/concept-fetch-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/502c2bfcf2e0b3705c6f3a059147d1b2/ui-colors-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.squareup.okhttp3/mockwebserver/3.11.0/df5ba87db2751d4076358ba6ad63045b031c8fd3/mockwebserver-3.11.0.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.squareup.okhttp3/okhttp-urlconnection/3.13.1/1bc9e51426e088b449f40a767e0c45c3baa910f3/okhttp-urlconnection-3.13.1.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.squareup.okhttp3/okhttp/3.14.1/67612a22d4b8f33c55263b188bf5a72774d06d18/okhttp-3.14.1.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jdk8/1.4.30/aeecfab2e6acbcb80fc6628e8882c98cbbd819cf/kotlin-stdlib-jdk8-1.4.30.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jdk7/1.4.30/cd81ed90fd79e43f587bd384c2bc6d1aed0bfe31/kotlin-stdlib-jdk7-1.4.30.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/e0b6eaa95ba01480767b49d629cd39fc/lifecycle-runtime-ktx-2.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/9b36eca580ed3ac5e40137955455f78c/room-ktx-2.2.5-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/05c9754dd1ccc8874aed33f8e40dfd9a/work-runtime-ktx-2.4.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-android/1.4.2/dbf36cc9f9cc03a72e20bf440cfeb89d74ca775f/kotlinx-coroutines-android-1.4.2.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/32caf75a1689328f1a738dd31a66e89c/palette-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter-params/5.3.1/9dfac3fbd6768974fc0c142304a3e90ba713b2a8/junit-jupiter-params-5.3.1.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter-engine/5.3.1/e2676b1786c57a80eb98f5bebd51a3d05e228c40/junit-jupiter-engine-5.3.1.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter-api/5.3.1/a7e97eac2784395cb991403f9641b042ad972941/junit-jupiter-api-5.3.1.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/2b1aed496448f118443ed5cc49783e33/core-testing-2.1.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/d788ba06e90cbbafee6dc6a38d4cb42d/rules-1.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/dc15269e58155ba29335f1db768dc4f2/runner-1.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.junit.vintage/junit-vintage-engine/5.3.1/48e21c1c1a7c1dde68c2f71181b3e86927c7923e/junit-vintage-engine-5.3.1.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/junit/junit/4.12/2973d150c0dc1fefe998f834810d68f278ea58ec/junit-4.12.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.mockito/mockito-core/2.24.5/599509fe319bd9e39559b8f987bee5d4b77167e4/mockito-core-2.24.5.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/cc4fbfba7f5a5eff2c7e366753d30e83/core-1.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/1563431e3c2c499c05587eb28e4c59e3/lifecycle-livedata-ktx-2.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/43a3927e1f6bff9b89834f9bba3715cb/lifecycle-livedata-core-ktx-2.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-core-jvm/1.4.2/4b9c6b2de7cabfb2c9ad7a5c709b1ddb7bbfd2ad/kotlinx-coroutines-core-jvm-1.4.2.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.4.30/d10d1e10f47006ee08162dde039e38ac487de4ac/kotlin-stdlib-1.4.30.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/93d6b82da9a85f9b3055d66a54e24635/legacy-support-core-ui-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/d93f5ca9ebd580feb9b2009b3f9c263f/legacy-support-core-utils-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/18711bc3a78085a287a5e5cf4c7b4214/media-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/b3481fbc7a4fc22b4962a0747292c26b/work-runtime-2.4.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/cb660cc123eb7fe2c79cd64b552d16f9/room-runtime-2.2.5-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/984312a471fc1eedc496337581e434d9/play-services-fido-18.1.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/512796b7bfb9664878f76e92b599f8ee/play-services-base-17.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/ebcbf2c20cf1ccaeb40424cfaacd4ab2/play-services-tasks-17.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/5723c50eee43d8b0e3d8a4cb2afa8985/play-services-basement-17.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/a0241c93ff4cb9f4aa71463066ec7bec/fragment-1.2.5-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/6d22062723de33e553ffb9745dd457e6/loader-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/c4aee73f11866a8ffede6a76c98b96c8/lifecycle-livedata-2.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/1780b302a2be3d59bbf022cce5b4fd15/activity-1.1.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/a9670e9ad558b00259a2ad6425718576/lifecycle-viewmodel-savedstate-2.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/e413433c3088bcf3ef885ba22433045a/lifecycle-livedata-core-2.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/467b187f0dc0dd449cbb2bbde31898ec/core-runtime-2.1.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/01260b6e2c7e8ab7c643392e066ea1fe/lifecycle-process-2.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/e2b7f7db0e689b90712b6b56029b35c1/swiperefreshlayout-1.1.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/ecde404a1099c2eff1fdd3fdd47521f1/viewpager-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/3e205ea674ce67741c7de0708907af1c/drawerlayout-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/9b256e1ba1f6cf0072277c8164ae67fd/slidingpanelayout-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/77eb5a9900498df4081a158e5863d70f/coordinatorlayout-1.1.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/081e591ffe533d9182f4fb7ebf3fa370/customview-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/6a77f855790766299f2e7b6e26149351/asynclayoutinflater-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/b872837d60b5033d436eeadf4f3a5d78/transition-1.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/c79cf521201e5d06fb87ff41151f600d/appcompat-resources-1.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/098b3f4f61704960aebd5ee2c6cd9cf6/vectordrawable-animated-1.1.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/6ee184a70ecec6b43045d1c16642fbb4/vectordrawable-1.1.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/d036d6a7fcde929bc5c0ab7bf1d8dd8c/core-1.3.2-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/0c2de9fedbaa38015faf7d245c92b029/lifecycle-service-2.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/1aaa4fae5a5cd85f0995f56d3f4fa70b/lifecycle-runtime-2.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/androidx.paging/paging-common/2.1.2/e8cdd44767173c4af5164732c32741c89afa684d/paging-common-2.1.2.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/87313bba480ea5b3ca5257ac3fe98887/savedstate-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/androidx.arch.core/core-common/2.1.0/b3152fc64428c9354344bd89848ecddc09b6f07e/core-common-2.1.0.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/5a3bad67327d80bbdaa8d2aaf3584dba/localbroadcastmanager-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/09b29464744bb3e631d53e87320932bb/monitor-1.3.0-rc03-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/a9b02cc9059a03b4fd1660af347737ec/documentfile-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/4b0effffe869956fda9fb2e1518b237c/print-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/fef930b32f9564dccacce8fc01a838b3/interpolator-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/9d7c7fa1ca625458caa5ca0778c3e0bc/cursoradapter-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/androidx.lifecycle/lifecycle-common-java8/2.0.0/52cb9efc663d8bebc82f8979a7da30938bd05611/lifecycle-common-java8-2.0.0.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/491766e7c251e8aedc260cf5b219001c/versionedparcelable-1.1.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/androidx.collection/collection/1.1.0/1f27220b47669781457de0d600849a5de0e89909/collection-1.1.0.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/androidx.concurrent/concurrent-futures/1.0.0/c1e77e3ee6f4643b77496a1ddf7a2eef1aefdaa1/concurrent-futures-1.0.0.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/androidx.room/room-common/2.2.5/f5e3b73a0c2ab5e276e26868e4ce3542baede207/room-common-2.2.5.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/3a815b8b196052753ff4a272261ffb14/lifecycle-viewmodel-2.2.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/androidx.lifecycle/lifecycle-common/2.2.0/4ef09a745007778eef83b92f8f23987a8ea59496/lifecycle-common-2.2.0.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/30a69b4e07c17315b815ba9bee2115e9/sqlite-framework-2.1.0-runtime.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/c347637f4c66eec533f4ef33946c2bfb/sqlite-2.1.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/androidx.annotation/annotation/1.1.0/e3a6fb2f40e3a3842e6b7472628ba4ce416ea4c8/annotation-1.1.0.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/io.sentry/sentry/1.7.21/830b5214d178189de482443aa20919697079edfd/sentry-1.7.21.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/778523868023ece6cb8b09593bf392ee/ui-icons-90.0.20210525143110-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.squareup.okio/okio/1.17.2/78c7820b205002da4d2d137f6f312bd64b3d6049/okio-1.17.2.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-engine/1.3.1/3ee68a06bbdab157dd260e2095c356481d6cd172/junit-platform-engine-1.3.1.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-commons/1.3.1/67b7edddfac1935e6e6d9b58d7c7df6db59b1d39/junit-platform-commons-1.3.1.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.apiguardian/apiguardian-api/1.0.0/3ef5276905e36f4d8055fe3cb0bdcc7503ffc85d/apiguardian-api-1.0.0.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.opentest4j/opentest4j/1.1.1/efd9f971e91074491ea55b19f67b13470cf4fcdd/opentest4j-1.1.1.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.hamcrest/hamcrest-core/1.3/42a25dc3219429f0e5d060061f71acb49bf010a0/hamcrest-core-1.3.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.robolectric/pluginapi/4.4/1ee94260f8c51620a35eac33fc1efc01350c751f/pluginapi-4.4.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.robolectric/shadowapi/4.4/529649474b53cf8f6f4a483044ade43aebed8a4c/shadowapi-4.4.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.robolectric/annotations/4.4/70fc5b1699467dfd7de606fc6c02ff9fc1816d9f/annotations-4.4.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/javax.inject/javax.inject/1/6975da39a7040257bd51d21a231b76c915872d38/javax.inject-1.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/javax.annotation/javax.annotation-api/1.3.2/934c04d3cfef185a8008e7bf34331b79730a9d43/javax.annotation-api-1.3.2.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.bouncycastle/bcprov-jdk15on/1.65/320b989112f00a63a3bcfa5a98f31a4f865a20fa/bcprov-jdk15on-1.65.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/net.bytebuddy/byte-buddy/1.9.7/8fea78fea6449e1738b675cb155ce8422661e237/byte-buddy-1.9.7.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/net.bytebuddy/byte-buddy-agent/1.9.7/8e7d1b599f4943851ffea125fd9780e572727fc0/byte-buddy-agent-1.9.7.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.objenesis/objenesis/2.6/639033469776fd37c08358c6b92a4761feb2af4b/objenesis-2.6.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/net.sf.kxml/kxml2/2.3.0/ccbc77a5fd907ef863c29f3596c6f54ffa4e9442/kxml2-2.3.0.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-common/1.4.30/bb9a3173350f55732416ee27956ea8f9b81f4dbb/kotlin-stdlib-common-1.4.30.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.jetbrains/annotations/13.0/919f0dfe192fb4e063e7dacadee7f8bb9a2672a9/annotations-13.0.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.slf4j/slf4j-api/1.7.24/3f6b4bd4f8dbe8d4bea06d107a3826469b85c3e9/slf4j-api-1.7.24.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-core/2.8.7/8b46f39c78476fb848c81a49fa807a9e9506dddd/jackson-core-2.8.7.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.yaml/snakeyaml/1.24/13a9c0d6776483c3876e3ff9384f9bb55b17001b/snakeyaml-1.24.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.google.guava/listenablefuture/9999.0-empty-to-avoid-conflict-with-guava/b421526c5f297295adef1c886e5246c39d4ac629/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/80e35b9b2b9bcec7600a88317a74bb79/annotation-experimental-1.0.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/androidx.constraintlayout/constraintlayout-solver/2.0.4/1f001d7db280a89a6c26b26a66eb064bb6d5efeb/constraintlayout-solver-2.0.4.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.jakewharton/disklrucache/2.0.2/93b399470e72c0bc8cb52379943b63c9aa586cd8/disklrucache-2.0.2.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/1bef3144ed1b8e5e8ac447bf5cb5f151/jna-5.6.0-runtime.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-commons/7.2/ca2954e8d92a05bacc28ff465b25c70e0f512497/asm-commons-7.2.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-util/7.2/a3ae34e57fa8a4040e28247291d0cc3d6b8c7bcf/asm-util-7.2.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-analysis/7.2/b6e6abe057f23630113f4167c34bda7086691258/asm-analysis-7.2.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-tree/7.2/3a23cc36edaf8fc5a89cb100182758ccb5991487/asm-tree-7.2.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm/7.2/fa637eb67eb7628c915d73762b681ae7ff0b9731/asm-7.2.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.almworks.sqlite4java/sqlite4java/0.282/745a7e2f35fdbe6336922e0d492c979dbbfa74fb/sqlite4java-0.282.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.ibm.icu/icu4j/53.1/786d9055d4ca8c1aab4a7d4ac8283f973fd7e41f/icu4j-53.1.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.google.auto.value/auto-value-annotations/1.6.2/ed193d86e0af90cc2342aedbe73c5d86b03fa09b/auto-value-annotations-1.6.2.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.google.guava/failureaccess/1.0.1/1dcf1de382a0bf95a3d8b0849546c88bac1292c9/failureaccess-1.0.1.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.checkerframework/checker-qual/2.5.2/cea74543d5904a30861a61b4643a5f2bb372efc4/checker-qual-2.5.2.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.google.errorprone/error_prone_annotations/2.2.0/88e3c593e9b3586e1c6177f89267da6fc6986f0c/error_prone_annotations-2.2.0.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/com.google.j2objc/j2objc-annotations/1.1/ed28ded51a8b1c6b112568def5f4b455e6809019/j2objc-annotations-1.1.jar:/Users/jalmeida/.gradle/caches/modules-2/files-2.1/org.codehaus.mojo/animal-sniffer-annotations/1.17/f97ce6decaea32b36101e37979f8b647f00681fb/animal-sniffer-annotations-1.17.jar:/Users/jalmeida/src/focus-android/app/build/intermediates/compile_and_runtime_not_namespaced_r_class_jar/focusDebug/R.jar:/Users/jalmeida/src/focus-android/app/build/intermediates/javac/focusDebug/classes:/Users/jalmeida/src/focus-android/app/build/tmp/kotlin-classes/focusDebug:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/31e56a097e4fc62a0ffa1fc11d089069/installreferrer-1.1/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/82d4b96b33befad79512c4299964c971/feature-contextmenu-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/9707f492c3537e0afb0e256fc76e30db/feature-app-links-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/0adb53459cd1450dff6140454386d1e6/feature-customtabs-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/cc3a4ee876613805cfbcc4e95e5f1a65/feature-intent-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/1c3e82fb01f6736dc02bdae6614048a7/feature-search-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/9563790f7ba2c314616c89f75f36aeac/feature-tabs-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/55a45db949e21619bdc899c1d8f23917/feature-session-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/e3c6f678b06c7dd9b497f487967fd568/browser-session-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/1e8a5c7b290e38815a090b044438bfc5/feature-downloads-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/41862494f2a5f79e617f8241ab7dfd24/feature-findinpage-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/8e455adb9c6f72f1448b76ad710bd8b5/feature-prompts-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/511d5c73ac2e91208a56bb91f14c4400/browser-toolbar-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/43abbfde3490f0d7fc8dcb24044c0957/browser-menu-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/d486296659bbaccc0770cef48bcbb706/browser-session-storage-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/4003059466598910d1aa18f44dd5bfcd/browser-thumbnails-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/a343424ebdbcfbb300d4109d63a09f34/browser-state-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/0b8844e28df41adbccf5fe2af1ea3421/browser-engine-gecko-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/6738156cc38ed9619bf152f5c68c2e76/concept-tabstray-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/252e14b4f1a0c5f5ad56c1cc9df0104d/concept-engine-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/e9c654544070f9f37237a814c9a1986d/lib-crash-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/9ea84685adfaedb2717d1cfea48a0900/lib-state-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/cce0af0cc74747849858e02af0fc9f6b/service-glean-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/863d95537d20bc9cd7d096c0ddab4fa0/service-location-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/609540216dffda8a953e2792a7b7151c/browser-errorpages-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/fc8afe27905acfc133d1521a8536e4ee/ui-tabcounter-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/881b48c91ee320ba3671cc489255c412/browser-menu2-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/05a2432c97263acd52860bb693638562/concept-menu-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/4580f7385ef8073c7093b50b636614e5/service-digitalassetlinks-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/ad216dbeedf93cd46fd08b9b36b36160/concept-toolbar-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/daf4bf7588ca4733352b1e0f16901b21/support-ktx-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/17bb3b66f77cec3d57693dd68848b3d1/ui-autocomplete-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/35f11394e4c9361b41471b61278f4ff1/support-utils-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/src/focus-android/service-telemetry/build/intermediates/library_java_res/debug/res.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/0279f60c2a780813133b7330a4f6f661/support-images-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/ec4d8873bb8d8fae34fedcf23bc35c05/support-base-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/a6884f259c1a6e98784c6137f1aa8b14/concept-storage-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/bf50c7a7796b144c85ab243f2584cad6/glean-38.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/cc0f2c8def8e7b3d02a02d96afcb6cfc/concept-base-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/e0b89432673094fe60dd8eca410725ba/preference-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/a77452e49f1b1a02c3fb9f968ee794f8/material-1.2.1/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/5afb07e94434c35f38a4072e015540ad/constraintlayout-2.0.4/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/1cebdd0cb62ee96ba6d82d4eb3252724/appcompat-1.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/ab1a9225d7ac271b545ceae199106624/browser-1.3.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/bd252a46c023f1b6a02aa1bf5ba425a5/core-ktx-1.3.2/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/7602d1e760abf229d4f523249ca44e44/cardview-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/d8566c2c2c13ea3fbd4ef660e85f5076/paging-runtime-2.1.2/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/9707758b67eb0793926d959df28e44bc/viewpager2-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/9a62a70274a70a80baadf3dfb668583a/recyclerview-1.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/22786bb75c49819f7d927d0cd4139184/geckoview-nightly-90.0.20210525093431/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/94a411c830ccfe277dd49a59e234f3f3/legacy-support-v4-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/b7cb06fdfe6e1124f5ab63806fc2d08e/lifecycle-extensions-2.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/9e8a892eef9149dd5a398891e15869a0/browser-domains-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/5510a99df70ab24e45f3dbff572a76f9/lib-fetch-okhttp-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/6649495d6f9e484ee388f3bff2b1dc14/concept-fetch-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/ceab4852ca75320d223c1fd444630644/ui-colors-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/6db00be38e8f5bcbb91e75774bdaea3d/lifecycle-runtime-ktx-2.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/e752397819bc17ef70419c54faea4cbc/room-ktx-2.2.5/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/f57e5567ba8179e752178d774f3f0e69/work-runtime-ktx-2.4.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/db39f25969e1df57f8a7c83999856e92/palette-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/cb75a1d2bfa101fbdcea7ecc7f454830/core-testing-2.1.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/55e92f6e25a6bb6f780651a499ec38aa/rules-1.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/1c7e44a41cb9952becc507daa816dcca/runner-1.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/86bc51f9344de2c733faad65ea4d12e3/core-1.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/8b91830be8a7fd3e5593f93a5884eeab/lifecycle-livedata-ktx-2.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/06a311992c275806754eafcbcf418722/lifecycle-livedata-core-ktx-2.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/bff5366f96ffdc3661c78657291e0ce7/legacy-support-core-ui-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/b24610df9369ff0969b2658d8450ce4d/legacy-support-core-utils-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/34ac03f8e1c63dc0859f87211b34c5d5/media-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/ac43a4b99ed8a4f71c3019bfdcd79c4c/work-runtime-2.4.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/8e03a9e366a2934f9befefd10c602917/room-runtime-2.2.5/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/c4d75a66d7693ec93a662a871c9a6845/play-services-fido-18.1.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/21e5aecd7492c0b8ef2fbc14b40b0eb0/play-services-base-17.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/12c81bcc65cb60319de0670fe274c572/play-services-tasks-17.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/93b6a12158fbe121d0710c9cd4160f42/play-services-basement-17.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/147175a6c56bc4e6e494b5265e007a4f/fragment-1.2.5/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/20188848923738ceb5ad11de5c5200b1/loader-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/ed2461abcd77b2425b399947ab2b1042/lifecycle-livedata-2.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/5540ac8b4098750209eba4fb31d834db/activity-1.1.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/0e20ad0108f5fb86d7cbe0c06b2d9ace/lifecycle-viewmodel-savedstate-2.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/b8d6d22b7d7ddf2668e4a9dd1acd1467/lifecycle-livedata-core-2.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/08362102e5bf5d5150c1ecc8db5f9e8c/core-runtime-2.1.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/6169c8b80960f217ff7b9626683531bd/lifecycle-process-2.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/d252d13fd7b33bb81ee566ce91b028c6/swiperefreshlayout-1.1.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/36de97678321d5eadd77afef7f67810c/viewpager-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/7640c04ce6d80049852812ee3aa6851d/drawerlayout-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/b76a316a99f7475c690654fe178a0fdb/slidingpanelayout-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/9dcf5f5c72568de51fdf8bd9a358e472/coordinatorlayout-1.1.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/78469aee7b254965b30e1956bfcbb25e/customview-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/5ac3ef0ca50830e4fa0e397329b286de/asynclayoutinflater-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/be0b8eb2d44cc1ccfbeae4480320e828/transition-1.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/79376807bce34fbd136eb820f35b8cd6/appcompat-resources-1.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/65e0da78bb9613fd85c1ee7149834ddf/vectordrawable-animated-1.1.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/816905beec122a9212c673214254a9c9/vectordrawable-1.1.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/7325fbff668234181bfb596cf97854d8/core-1.3.2/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/04ef2866830ad118bf92f4cf3e9d91a5/lifecycle-service-2.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/8894fd351bfc682f394ffc6fdf7c016c/lifecycle-runtime-2.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/ea2a9331c37d9f6d11758b5e50bbfa27/savedstate-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/30f2ce583cdb1e061e2d70c70f55c4c5/localbroadcastmanager-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/d2e93e7a0c6e8fda2a5d657b6a963ee5/monitor-1.3.0-rc03/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/839c7bb928a6a087500b514c0161ff97/documentfile-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/b62b5a7b68556024837bd4c5bac5078c/print-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/0cbc3bbf668d2a804bca0284e59ed477/interpolator-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/c2fd7ca49c0f447223c66b875c2b40fd/cursoradapter-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/adf81b82f94a0cd15e3db709beb6af60/versionedparcelable-1.1.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/3e27cfdac886e7b9d1eeaf437b695a74/lifecycle-viewmodel-2.2.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/6a40dc0ca777ecbec5ddeb18ecb8b901/sqlite-framework-2.1.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/8c5edc9e1e8e4f58b71019f63901edef/sqlite-2.1.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/5bec7fdc24bc3e01917a19dd0ad49afd/ui-icons-90.0.20210525143110/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/dcc6b72b51bd90a58c813a03f8ba9546/annotation-experimental-1.0.0/jars/classes.jar:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/29509c42b190e288c362e63e49a0b4b3/jna-5.6.0/jars/classes.jar:/Users/jalmeida/src/focus-android/app/build/intermediates/java_res/focusDebug/out:/Users/jalmeida/.gradle/caches/transforms-2/files-2.1/f5e15ff62d155b5e1b4fe2da328bb96c/android.jar)
	at com.sun.jna.Native.loadNativeDispatchLibraryFromClasspath(Native.java:1032)
	at com.sun.jna.Native.loadNativeDispatchLibrary(Native.java:988)
	at com.sun.jna.Native.<clinit>(Native.java:195)
	at com.sun.jna.StringArray.<init>(StringArray.java:56)
	at com.sun.jna.StringArray.<init>(StringArray.java:49)
	at mozilla.telemetry.glean.private.PingType.<init>(PingType.kt:55)
	at org.mozilla.focus.GleanMetrics.Pings.<clinit>(Pings.kt:32)
	at org.mozilla.focus.telemetry.GleanMetricsService.initialize(GleanMetricsService.kt:67)
	at org.mozilla.focus.FocusApplication.onCreate(FocusApplication.kt:49)
```

Reply:
Looks like @badboy already responded to that one, so it should be covered now :-)